internal/deps/sisu_test.go

```diff
@@ -1,11 +1,11 @@
-package main
+package deps
 
 import (
 	"testing"
 )
 
 func TestCompositionFromSisuDir(t *testing.T) {
-	comp, err := compositionFromSisuDir("testdata")
+	comp, err := CompositionFromSisuDir("testdata", "stage", "eu")
```

Comment:
do you mean staging?

Reply:
nice catch, that should be "stg" because the file in testdata/ is named `.deps-stg-eu.toml`

Reply:
fixed, also the other occurence in that file